### PR TITLE
Allow creation of Apply 2 applications

### PR DESCRIPTION
**Pull request by fofr**
- Include an "apply2" boolean for each application, which we can use to toggle Apply2 specific features (eg limit that application to 1 course choice)
- Create an apply2 dummy application derived from the existing dummy
- Add a route for copying an existing application and making it into a new Apply 2 one
- Add missing suitability fields to defaults and dummy applications

### CODE REVIEW

**Comment by paulrobertlloyd** [comment]
Should the `apply2` boolean be a feature flag?

File: app/data/dummy-apply-2-application.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Copies dummy application and tweaks some fields
+var dummyApply2Application = JSON.parse(JSON.stringify(require('./dummy-application')))
+
+// Custom Apply 2 fields
+dummyApply2Application.apply2 = true
+dummyApply2Application.previousApplications = ['12345']
+
+// No sections completed
+dummyApply2Application.completed = {}
```
Comment (paulrobertlloyd) [inline app/data/dummy-apply-2-application.js]: `completed` or `reviewed`. Same to ensure parity with Apply 1 application?

File: app/data/dummy-apply-2-application.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Copies dummy application and tweaks some fields
+var dummyApply2Application = JSON.parse(JSON.stringify(require('./dummy-application')))
```
Comment (paulrobertlloyd) [inline app/data/dummy-apply-2-application.js]: Smart!

File: app/data/dummy-apply-2-application.js (hunk 1 above)
Comment (fofr) [reply app/data/dummy-apply-2-application.js]: I was going to use the same data structure but change the language in the UI, so we reuse logic.

**Pull request merged**